website/docs/r/default_subnet.html.markdown

```diff
@@ -40,6 +40,7 @@ The arguments of an `aws_default_subnet` differ slightly from those of [`aws_sub
 The following additional arguments are supported:
 
 * `force_destroy` - (Optional) Whether destroying the resource deletes the default subnet. Default: `false`
+* `tags` - (Optional) Key-value mapping of resource tags. If configured with a provider [`default_tags` configuration block](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#default_tags-configuration-block) present, tags with matching keys will overwrite those defined at the provider-level.
```

Comment:
Since `tags` is documented in the subnet resource, it may be misleading to include in this section on additional arguments.
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/subnet#tags

Reply:
@jar-b Good catch. I've removed the change.